Remotion/Web/Development.WebTesting.TestSite.Shared/TreeViewTest.aspx.cs

```diff
@@ -79,7 +79,7 @@ private void InitializeTreeView ()
 
     private void InitializeMyTreeView ()
     {
-      var root = new TreeNode("Root node", "RootValue", "~/Images/SampleIcon.gif");
+      var root = new TreeNode("Root node", "RootValue", this.ResolveImageResource("SampleIcon.gif"));
```

Comment:
Would this also work with "Image/SampleIcon.gif"?
Not sure we should switch this, since we are in code behind and "Image" is a magic value that's normally resolved via the ResourceType. So, it's more a question of interest than a request for change :)

Unless we could completly revert the ResolveImageResource usage and drop the helper API again?

Reply:
Yes would also work without. We still have a lot of usages of the helper API so I would not drop it again. The repeating the `IResourceUrlFactory` is 🤢

Reply:
Yeah, it's only a good idea if we can get rid of all the usages. At least those for image, though the benefit of that is already pretty small, so, okay, keep it.